docs/network-flow-visibility.md

```diff
@@ -678,26 +713,47 @@ spec:
       clickhouse_operator/networks/ip: "::/0"
 ```
 
-ClickHouse credentials are also specified in [flow-aggregator.yml][flow_aggregator_manifest_yaml]
-as a resource of kind: a Secret. Please also make the corresponding changes.
+ClickHouse credentials are also specified in `flow-aggregator.yml` as a Secret
+named `clickhouse-secret` as shown below. Please also make the corresponding changes.
 
-Grafana login credentials are specified in [grafana.yml][grafana_manifest_yaml] as
-resource of kind: a Secret.
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: flow-aggregator
+  name: clickhouse-secret
+  namespace: flow-aggregator
+stringData:
+  password: clickhouse_operator_password
+  username: clickhouse_operator
+type: Opaque
+```
 
-We recommend changing all the credentials above if you are going to run the Flow Collector
-in production. After making any credentials change, run the following command to generate
-a new manifest:
+Grafana login credentials are specified in `flow-visibility.yml` as a Secret named
+`grafana-secret`.
 
-```shell
-make manifest
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: grafana-secret
+  namespace: flow-visibility
+stringData:
+  admin-password: admin
+  admin-username: admin
+type: Opaque
 ```
 
+We recommend changing all the credentials above if you are going to run the Flow
+Collector in production.
+
 ##### ClickHouse Configuration
 
 The ClickHouse database can be accessed through the service `clickhouse-clickhouse`.
```

Comment:
"service" refers to K8 Service? If so, use "Service" and change other occurrences.